Ports/milkytracker/patches/tracker-CMakeLists.patch

```diff
@@ -0,0 +1,10 @@
+--- MilkyTracker-1.03.00d/src/tracker/CMakeLists.txt	2021-01-04 15:49:14.173940007 +0100
++++ MilkyTracker-1.03.00/src/tracker/CMakeLists.txt	2021-01-04 15:48:03.179731481 +0100
+@@ -374,4 +375,7 @@
+     set(INSTALL_DEST ${CMAKE_INSTALL_BINDIR})
+ endif()
+ 
++target_link_libraries(tracker -L../../SDL2/SDL-master-serenity/ -lSDL2 -lgui -lipc -lm)
```

Comment:
We should link with the libSDL in the `Build/Root/` subtree, not with the one from the `Ports/SDL2/` subtree.